transforms/language/lang_id/python/src/lang_id_transform.py

```diff
@@ -53,7 +57,9 @@ def __init__(self, config: dict[str, Any]):
         self.nlp_langid = LangModelFactory.create_model(
             config.get(model_kind_key), config.get(model_url_key), config.get(model_credential_key)
         )
-        self.column_name = config.get(content_column_name_key)
+        self.content_column_name = config.get(content_column_name_key)
```

Comment:
You should not assume the config map will come from the CLI and apply_input_params() below.  As a result, you really should check for missing keys here and/or provide the defaults as appropriate.

Reply:
Oh, ok. I'll set default value.

Reply:
Looks good.  One minor point, can you define the default values ("score", "content"0, which are used both in init() and in add_input_params(), using global variables and then reference those in the two methods?  A bit of extra indirection, but less prone to developer error at a later point in time. (I'm trying to promote this pattern with others as well).

Reply:
Ok, I'll define const.